setup-guide.md

```diff
@@ -64,5 +64,412 @@ $git config --global core.autocrlf true
+}
+```
+
+### dotenv
+
+#### 16.3.1
+
+Clean upgrade. No issues.
+
+```json
+"devDependencies": {
+  "dotenv": "^16.3.1",
+  ...
+}
+```
+
+### eslint-plugin-vue
+
+#### 9.17.0
+
+Clean upgrade. No issues.
+
+```json
+"devDependencies": {
+  "eslint-plugin-vue": "^9.17.0",
+  ...
+}
+```
+
+### sass
+
+#### 1.69.2
+
+Clean upgrade. No issues.
+
+```json
+"devDependencies": {
+  "sass": "^1.62.0",
+  ...
+}
+```
+
+### yaml
+
+#### 2.3.2
+
+Clean upgrade. No issues.
+
+```json
+"devDependencies": {
+  "yaml": "^2.3.2",
+  ...
+}
+```
+
+### webpack
+
+#### 5.88.2
+
+Clean upgrade. No issues.
+
+```json
+"devDependencies": {
+  "webpack": "^5.88.2",
+  ...
+}
+```
+
+### vue-tsc
+
+#### 1.8.19
+
+Clean upgrade. No issues.
+
+```json
+"devDependencies": {
+  "vue-tsc": "^1.8.19",
+  ...
+}
+```
+
+### typescript
+
+#### 5.2.2
+
+```json
+"devDependencies": {
+  "typescript": "^5.2.2",
+  ...
+}
+```
+
+When we upgraded the Typescript version to 5.2.2 there was a conflict with
+eslint. You can see the problem below.
+
+```log
+=============
+
+WARNING: You are currently running a version of TypeScript which is not
+officially supported by @typescript-eslint/typescript-estree.
+
+You may find that it works just fine, or you may not.
+
+SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <5.1.0
+
+YOUR TYPESCRIPT VERSION: 5.2.2
+
+Please only submit bug reports when using the officially supported version.
+
+=============
+```
+
+### Types/node
+
+#### 20.8.4
+
+```json
+"devDependencies": {
+  "@types/node": "^20.8.4",
+  ...
+}
+```
+
+This version of type/node does not work with version 3.4.1 of nuxt. You can
+downgrade the version or upgrade the version of nuxt for a solution.
+
+### Nuxt
+
+#### 3.7.4
+
+With this version, the generate process fails if the url of the links cannot be
+generated. So make sure that the target page of the links exists.
+
+### Nuxt Content
+
+#### 2.5.2
+
+[nuxt.config.ts]
+```ts
+content: {
+  anchorLinks: {
+    depth: 0
+  }
+}
+```
+
+#### 2.8.5
+
+[nuxt.config.ts]
+```ts
+mdc: {
+  headings: {
+    anchorLinks: {
+      h1: false,
+      h2: false,
+      ...
+    }
+  }
+}
+```
+
+## Move new project
+
+First create new nuxt project with latest version
+
+`npx nuxi@latest init .theme`
+
+Replace to this code in `app.vue`
+
+```vue
+<template>
+  <NuxtLayout name="default">
+    <NuxtPage />
+  </NuxtLayout>
+</template>
+```
+
+Create new page `[...content-page]` and add this code to under `/pages` directory
+
+```vue
+<template>
+  <ContentDoc />
+</template>
+```
+
+Add content module
+
+`npm install @nuxt/content`
+
+Configure `nuxt.config.ts`
+
+```ts
+export default defineNuxtConfig({
+  devtools: { enabled: true },
+  modules: [
+    '@nuxt/content'
+  ],
+  content: {
+    // ... options
+  }
+})
+```
+
+If you have come this far, create a markdown file called `index.md` under the
+`/content` folder. Display your page by saying `npm run dev`.
+
+Make this configuration to prevent the headers from being in link format
```

Comment:
better to point out that this change is required due to a breaking change saying something like;

`There is a breaking change in configuration when disabling links for header.` and then show how it was before, and then show how it should be now.